README.Rmd

```diff
@@ -15,6 +15,16 @@ knitr::opts_chunk$set(
 pkgload::load_all()
 
 set.seed(20230702)
+
+clean_output <- function(x, options) {
+  gsub("0x[0-9a-f]+", "0xdeadbeef", x)
```

Comment:
I used the same function as in https://github.com/duckdblabs/duckplyr/blob/13e5d48551982d140bff36de646c7dfdf3602532/tests/testthat/test-relational-duckdb.R#L99

I had never heard of 0xdeadbeef before.